packages/utils/src/validation/error.ts

```diff
@@ -0,0 +1 @@
+export type TransactionErrorKey = 'InvalidAddress' | 'InsufficientFunds' | 'InvalidNetworkAddress';
```

Comment:
Adding this to share keys with `Stacks`

Reply:
I'm not sure details like this should go in utils, in favour of utils being a home for js helpers, and more generic, non-domain specific code.

Could we have a ValidationService, or something?

Reply:
I'll move it to the bitcoin package now as that's the only place using it. I could add it to `models`? I just want to share the type across packages.

I can investigate a validation service. I have a few other functions to migrate but I think thinks like `isValidPrecision` can live in `utils` and we may not need a full service.